.changeset/famous-doors-chew.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": minor
+"@redocly/cli": minor
+---
+
+Fixed Arazzo schema mismatch.
```

Comment:
What was mismatched?

Reply:
Fix: workflows input wrong type
Fix: type inconsistency for parameter object
Fix: criteria type is wrong

Reply:
```suggestion
Updated the Arazzo validation types for workflows input, parameter objects, and criteria to match the specification.
```